apps/ios/SimpleXChat/ChatTypes.swift

```diff
@@ -117,6 +117,181 @@ extension NamedChat {
 
 public typealias ChatId = String
 
+public struct FullPreferences: Decodable {
```

Comment:
maybe it's better to define defaults in types instead of views

Reply:
They were just examples, there should be no defaults in UI, they are set from api responses